ubtue_Revista electronica de ciencia penal y criminologia.js

```diff
@@ -0,0 +1,176 @@
+{
+	"translatorID": "7e638a55-f469-4324-89c9-e31aa71c4b46",
+	"label": "ubtue_Revista electrónica de ciencia penal y criminología",
+	"creator": "Johannes Riedl",
+	"target": "^https?://criminet.ugr.es/recpc/\\d+",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2020-10-21 16:05:51"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2020 Universitätsbibliothek Tübingen.  All rights reserved.
+
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+const entriesXPath = '//p[@class="MsoFooter"]';
+const journalInfoXPath = '//p[@class="MsoNormal"]//span/text()';
+
+const articleNumberPrefix = '^\\s*\\d+-[a-z]?\\d+[a-z]?';
+const quotationMarks = '["”]';
+
+function detectWeb(doc, url) {
+	if (getSearchResults(doc, true)) {
+		return "multiple";
```

Comment:
No single mode?

Reply:
Would not make sense in this context: Individual items are generated from the webpage with the whole issue.